<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setDialog</class>
 <widget class="QDialog" name="setDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="bucketLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bucket Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="urlLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Url</string>
   </property>
  </widget>
  <widget class="QLabel" name="secretKeyLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SecretKey</string>
   </property>
  </widget>
  <widget class="QLabel" name="accessKeyLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AccessKey</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bucketLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>25</y>
     <width>461</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>55</y>
     <width>461</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="accessKeyLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>85</y>
     <width>461</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="secretKeyLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>115</y>
     <width>461</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Ok</string>
   </property>
  </widget>
  <zorder>okButton</zorder>
  <zorder>bucketLabel</zorder>
  <zorder>urlLabel</zorder>
  <zorder>secretKeyLabel</zorder>
  <zorder>accessKeyLabel</zorder>
  <zorder>bucketLineEdit</zorder>
  <zorder>urlLineEdit</zorder>
  <zorder>accessKeyLineEdit</zorder>
  <zorder>secretKeyLineEdit</zorder>
  <zorder>cancelButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
